### DISCUSSION TITLE
TLS 1.0 not supported by stripe

### DISCUSSION THREAD

**Discussion by gusreyes01** [pinax/pinax-stripe#654] [Show and tell]
I'm using  **python==2.7.11**, **pinax-stripe==3.2.1** with **Django==1.9.6** and getting the following error when running: `python manage.py sync_plans
`

`stripe.error.AuthenticationError: Request req_8x3dWCHUrqcqGs: Stripe no longer supports API requests made with TLS 1.0. Please initiate HTTPS connections with TLS 1.2 or later. You can learn more about this at https://stripe.com/blog/upgrading-tls.
`

**Comment by gusreyes01** [comment]
This seems more an issue of the python-stripe package.

**Comment by gusreyes01** [comment]
Solved it after installing this libraries:

pip install urllib3
pip install pyopenssl
pip install ndg-httpsclient
pip install pyasn1

**Comment by irothschild** [comment]
In case anyone else is having the same problem, I had to uninstall and reinstall the `requests` module after installing the 4 modules @gusreyes01 listed above.

**Comment by johndark** [comment]
Got same issue today with no longer supports TLS 1.0. Just updated mine Python (windows) from 2.7.2 to 2.7.12 and it works well now.

**Comment by paltman** [comment]
@gusreyes01 @irothschild @johndark thanks for the information.  not sure what the solution is yet to make it _just work_ for folks but at least this thread is a start for people to fix it until we figure out what the solution is.

**Comment by leehinde** [comment]
@gusreyes01 Thanks! That worked for me. I thought I was going to have to do much more based on the url that Strip provides.

Now I have to go see what all those packages do. I assume they forward all the cc data to you. :-)

**Comment by studiorooster** [comment]
@gusreyes01 Worked for me!! Thanks!

**Comment by rguisando** [comment]
How about stripe.net? Not working even after upgraded to 8.0.0

**Comment by Jony-Y** [comment]
@gusreyes01 LIFE SAVER! thanks

**Comment by paltman** [comment]
We should summarize this workaround/scenario in the documentation somewhere before closing this.

**Comment by mesas998** [comment]
I updated to python 3.6.3.
pip3.6 is broke but pip3 works.
I did the following:
$ pip3 install urllib3
$ pip3 install pyopenssl
$ pip3 install ndg-httpsclient

Still getting "TLS 1.0 not supported by stripe"
